package.json

```diff
@@ -18,8 +20,12 @@
   },
   "homepage": "https://github.com/LambdaSchool/Sprint-Challenge-Node-Express#readme",
   "dependencies": {
+    "express": "^4.16.4",
     "helmet": "^3.15.1",
     "knex": "^0.16.3",
     "sqlite3": "^4.0.6"
+  },
+  "devDependencies": {
+    "nodemon": "^1.18.10"
```

Comment:
Nice on putting nodemon in the devDependencies rather than the regular dependencies!